lib/internal/encoding.js

```diff
@@ -443,6 +446,10 @@ function makeTextDecoderICU() {
         return decodeUTF8(input, this[kIgnoreBOM], this[kFatal]);
       }
 
+      if (this[kLatin1FastPath]) {
+        return decodeLatin1(input);
```

Comment:
You should support ignore bom, and fatal values as well.

Reply:
I made an update covering this, thank you